src/default-config.ls

```diff
@@ -0,0 +1,53 @@
+require! <[ path fs ]>
+
+{ filter, map, first, join, keys } = require 'prelude-ls'
+
+# RC automatically overwrites these with env variables.
+# For example to edit environment set arch_environment
+
+/* Map of parsers which take a file path and parse functions*/
+
+parsers =
+  js: -> require it,
+  ls: -> require it
+
+parser = (fname) -> parsers[(path.extname fname).slice(1)](fname)
+
+fpath-regex = new RegExp "arch\.config\.(?:#{parsers |> keys |> join '|'})$"
+
+filter-configs = -> fpath-regex.test it
+
+merge = (x, xs) -> x import xs
+
+initial-conf =
+  app-path:     process.env.arch_app_path or path.resolve '.'
```

Comment:
Ok, how about this: instead of using a host of env variables to override the configuration, why don't we allow _one_ with a JSON config, specifically for the CLI to use and not "exposed" to the rest of the world - i.e. "internal only, use at your own risk"?